spec/controllers/curation_concerns/generic_works_controller_json_spec.rb

```diff
@@ -66,10 +73,12 @@
         expect(response.location).to eq main_app.curation_concerns_generic_work_path(created_resource)
       end
     end
+
     describe 'failed update' do
-      before { post :update, id: resource, generic_work: { title: nil, depositor: nil }, format: :json }
+      before { post :update, id: resource, generic_work: { title: [] }, format: :json }
```

Comment:
I think the tests in hydra-derivatives and in `spec/forms/work_form_spec.rb` cover this.